jsonparser/switcher.go

```diff
@@ -0,0 +1,129 @@
+package jsonparser
+
+import (
+	"github.com/rudderlabs/rudder-go-kit/config"
+)
+
+const (
+	// TidwallLib is the implementation using gjson/sjson libraries
+	TidwallLib = "tidwall"
+	// GrafanaLib is the implementation using jsonparser library
+	GrafanaLib = "grafana"
+	// DefaultLib is the default implementation
+	DefaultLib = TidwallLib
```

Comment:
Am I reading the benchmarks wrong? It seems to be that the grafana one is faster in all 3 scenarios, get, set and delete. It often does 0 allocations as well.